thirdparty/build-thirdparty.sh

```diff
@@ -638,6 +638,8 @@ build_gperftools() {
         ./autogen.sh
     fi
 
+    patch -p1 < "$TP_PATCH_DIR/gperftools_20251105.patch"
```

Comment:
patch operation shall be part of `download_thirdparty.sh`

Reply:
fixed